interfaces/devdoc/threadpool_requirements.md

```diff
@@ -104,6 +110,60 @@ MOCKABLE_FUNCTION(, void, threadpool_close, THREADPOOL_HANDLE, threadpool);
 
 **NON_THREADPOOL_01_019: [** If `threadpool` is not OPEN, `threadpool_close` shall return. **]**
 
+### threadpool_create_work_item
+
+```c
+MOCKABLE_FUNCTION(, PVOID, threadpool_create_work_item, THANDLE(THREADPOOL), threadpool, THREADPOOL_WORK_FUNCTION, work_function, PVOID, work_function_context);
+```
+
+`threadpool_create_work_item` creates a work item to be executed by the threadpool.
```

Comment:
>uted by the threadpool. [](http://example.com/codeflow?start=60&length=23)

Here or somewhere we should document *why* we have this separate API. Like "This function separates the creation of the work item from scheduling it so that `threadpool_schedule_work_item` can be called later and cannot fail."